test/built-ins/Number/prototype/toFixed/return-values.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2016 The V8 Project authors. All rights reserved.
```

Comment:
Does v8 own the copyright to this file? If not, the copyright line must change.

The proper thing to do is to sign the CLA and put your own name there.